packages/material-ui/src/index.js

```diff
@@ -401,5 +401,4 @@ export * from './GlobalStyles';
 /**
  * @deprecated will be removed in v5 beta, please use StyledEngineProvider instead
  */
-export { default as StylesProvider } from './StyledEngineProvider';
```

Comment:
Should we still keep it?

Reply:
I think that it would be confused to have StylesProvider with emotion and StylesProvider with JSS in the documentation. 👍 for removing it.